clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1892,6 +1892,37 @@ public Map<TopicPartition, OffsetAndMetadata> committed(final Set<TopicPartition
     }
 
     /**
+     * Determines the client's unique client instance ID used for telemetry. This ID is unique to
+     * this specific client instance and will not change after it is initially generated.
+     * The ID is useful for correlating client operations with telemetry sent to the broker and
+     * to its eventual monitoring destination(s).
```

Comment:
There's no need for the "(s)" in the comment. It's already entirely readable.

Reply:
Removed (s) but kept destinations as there could be multiple.